tests/test_checks.py

```diff
@@ -236,8 +239,43 @@ def test_check_w007_invalid(self, mocked_guess_type):
             ],
         )
 
+    def test_debug_toolbar_installed_when_running_tests(self):
+        with self.settings(DEBUG=False):
+            dt_settings.get_config()["IS_RUNNING_TESTS"] = False
+            errors = debug_toolbar_installed_when_running_tests_check(None)
+            self.assertEqual(len(errors), 0)
+        with self.settings(DEBUG=True):
+            dt_settings.get_config()["IS_RUNNING_TESTS"] = True
+            errors = debug_toolbar_installed_when_running_tests_check(None)
+            self.assertEqual(len(errors), 0)
+        with self.settings(DEBUG=True):
+            dt_settings.get_config()["IS_RUNNING_TESTS"] = False
+            errors = debug_toolbar_installed_when_running_tests_check(None)
+            self.assertEqual(len(errors), 0)
+        with self.settings(DEBUG=False):
+            dt_settings.get_config()["IS_RUNNING_TESTS"] = True
+            errors = debug_toolbar_installed_when_running_tests_check(None)
+            self.assertEqual(
```

Comment:
If I had written this I would have written it as `self.assertEqual([error.id for error in errors], ["debug_toolbat.W009"])` so that the test doesn't have to change if we reword the error itself for some reason. But I'm fine with your more strict version.